module/mempool/herocache/backdata/heropool/pool_test.go

```diff
@@ -597,7 +598,7 @@ func testAddingEntities(t *testing.T, pool *Pool, entitiesToBeAdded []*unittest.
 	lruEjectedIndex := 0
 	for i, e := range entitiesToBeAdded {
 		// adding each element must be successful.
-		entityIndex, slotAvailable, ejectedEntity := pool.Add(e.ID(), e, uint64(i))
+		entityIndex, slotAvailable, ejectedEntity, _ := pool.Add(e.ID(), e, uint64(i))
```

Comment:
we shouldn't ignore returned values, better to add an assertion for expected value.